pkg/resource/testdata/TestStatefulSetBuilder/insecure_statefulset_with_resources.golden

```diff
@@ -73,6 +73,11 @@ spec:
             scheme: HTTP
           initialDelaySeconds: 10
           periodSeconds: 5
+        lifecycle:
+          preStop:
```

Comment:
Naive question, is the 1m grace period the default or set somewhere else, should we be setting it explicitly here?

Reply:
we need to add `terminationGracePeriodSeconds` and set it to 60 seconds here good catch @udnay